### PR TITLE
Add instructions to quickstart guide to create and cd into a directory

### PR DESCRIPTION
**Pull request by valaparthvi**
Signed-off-by: Parthvi Vala <pvala@redhat.com>

**What type of PR is this:**
/kind bug
/area documentation

**What does this PR do / why we need it:**
This PR adds instructions to create and cd into a new directory before running any odo commands.

**Which issue(s) this PR fixes:**

Fixes #

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [x] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

**Review by rm3l — COMMENTED** [review]
LGTM.
While we're in the Quickstart guides, can you also add a note in the .NET guide (in `Step 2. Initializing your application (odo init)`) stating that odo will detect .NET 5.0, and that, depending on the Dotnet SDK used to create the project, the user might need to answer "No" to the `odo init` prompt and pick the appropriate .NET version (like .NET 6.0 in our example) instead.
Otherwise, `odo dev` will fail later while running the build command.
I found it not clear enough as it is a bit hidden in the sample output of `odo init`.

Or, if you prefer, this can be done in a subsequent PR.

File: docs/website/docs/user-guides/quickstart/docs-mdx/odo_init_description.mdx [hunk 1]
```diff
@@ -6,6 +6,10 @@ Now we'll initialize your application by creating a `devfile.yaml` to be deploye
 
 <p>Let's run <code>odo init</code> and select <span>{props.framework}</span>:</p>
 
+:::note
+When you first run `odo init`, it will detect the required devfile to be 'dotnet50', if this happens to you, please select <b>No</b> when asked <b>Is this correct?</b> and then select <b>.NET 6.0</b> when asked for <b>Select project type:</b>. Take a look at the sample output below for a reference.
+:::
```
Comment (rm3l) [inline docs/website/docs/user-guides/quickstart/docs-mdx/odo_init_description.mdx:11]: Adding the note here will affect all the other Quickstart Guides, not only the one for . NET.

**Review by rm3l — CHANGES_REQUESTED** [review]

File: docs/website/docs/user-guides/quickstart/docs-mdx/odo_init_description.mdx (hunk 1 above)
Comment (valaparthvi) [reply docs/website/docs/user-guides/quickstart/docs-mdx/odo_init_description.mdx:11]: :facepalm: You're right.

**Comment by rm3l** [comment]
/override OpenShift-Integration-tests/OpenShift-Integration-tests

Doc-only PR.

**Pull request merged**